lua/nordic/groups/integrations.lua

```diff
@@ -523,6 +524,24 @@ function M.get_groups()
     G.WhichKeySeperator = {}
     G.WhichKeyValue = {}
 
+    -- Markdown.Nvim.
+    G.RenderMarkdownH1 = { link = '@markup.heading.1' }
+    G.RenderMarkdownH2 = { link = '@markup.heading.2' }
+    G.RenderMarkdownH3 = { link = '@markup.heading.3' }
+    G.RenderMarkdownH4 = { link = '@markup.heading.4' }
+    G.RenderMarkdownH5 = { link = '@markup.heading.5' }
+    G.RenderMarkdownH6 = { link = '@markup.heading.6' }
+    G.RenderMarkdownH1Bg = { bg = U.blend(C.yellow.base, C.grey0, 0.3) }
+    G.RenderMarkdownH2Bg = { bg = U.blend(C.orange.base, C.grey0, 0.3) }
+    G.RenderMarkdownH3Bg = { bg = U.blend(C.magenta.base, C.grey0, 0.3) }
+    G.RenderMarkdownH4Bg = { bg = U.blend(C.green.base, C.grey0, 0.3) }
+    G.RenderMarkdownH5Bg = { bg = U.blend(C.blue2, C.grey0, 0.3) }
+    G.RenderMarkdownH6Bg = { bg = U.blend(C.cyan.base, C.grey0, 0.3) }
```

Comment:
This should blend with `C.bg` because we have multiple background colors and the user can specify their own.

```suggestion
    G.RenderMarkdownH1Bg = { bg = U.blend(C.yellow.base, C.bg, 0.3) }
    G.RenderMarkdownH2Bg = { bg = U.blend(C.orange.base, C.bg, 0.3) }
    G.RenderMarkdownH3Bg = { bg = U.blend(C.magenta.base, C.bg, 0.3) }
    G.RenderMarkdownH4Bg = { bg = U.blend(C.green.base, C.bg, 0.3) }
    G.RenderMarkdownH5Bg = { bg = U.blend(C.blue2, C.bg, 0.3) }
    G.RenderMarkdownH6Bg = { bg = U.blend(C.cyan.base, C.bg, 0.3) }
```

Also I think the blend strength should be lower (0.15) but that's just personal preference.

Reply:
Sounds good. I 'stole' the .3 from cattpuuchin's implementation, but agree .15 looks better.